README.md

```diff
@@ -62,12 +62,76 @@ Use `action-validator -h` to see additional options.
 > repo -- or, heaven forfend, outside the repository entirely.
 
 
+## Pre-commit hook example
+
+Create an executable file in the .git/hooks directory of the target repository:
+`touch .git/hooks/pre-commit && chmod +x .git/hooks/pre-commit` and paste the following example code:
+
+```bash
+#!/bin/bash
+if [ -d ".github/workflows" ]; then
+  if command -v action-validator >/dev/null 2>&1; then
+    echo "Running pre-commit hook for GitHub Actions: https://github.com/mpalmer/action-validator"
+    scan_count=0
+    for action in $(find .github/workflows -name "*.y*ml"); do
+      validate="$(action-validator "$action")"
+      if [ -z "$validate" ]; then
+        echo "✅ $action"
+      else
+        echo "❌ $action"
+        echo "$validate"
+        exit 1
+      fi
+      scan_count=$((scan_count+1))
+    done
+    echo "action-validator scanned $scan_count GitHub Actions found no errors!"
+  else
+    echo "action-validator is not installed!"
+    echo "Installation instructions at: https://github.com/mpalmer/action-validator"
+    echo "Skipping GitHub Action linting..."
```

Comment:
This is a nice touch, although given that git hooks are repo-local, what are the chances that someone's going to install this hook and then *not* want to actually run it?

Reply:
This would bite me all the time actually. I will often use a [docker container with all of the tools I use installed](https://github.com/mathew-fleisch/docker-dev-env), and mount my mac's home folder with source code on it. I suppose it is better not do the test, and just let the hook fail though.

Reply:
Fair enough, I hadn't considered that use case.  I suppose if people want to be stricter, they can change the "Skipping" line to `exit 42`; it's their hook script, after all!